src/hooks/useTitle.js

```diff
@@ -4,8 +4,8 @@ import {useTitle as reactUseTitle} from 'react-use';
 import {ConfigContext} from 'Context';
 
 const useTitle = localTitle => {
-  const {titlePrefix} = useContext(ConfigContext);
-  return reactUseTitle(titlePrefix ? `${titlePrefix} | ${localTitle}` : localTitle);
+  const {documentTitle} = useContext(ConfigContext);
```

Comment:
IMO a name like `titleSuffix` or `baseTitle` is more appropriate, because the point of this hook is setting the document title (`document.title`) so this could be confusing?

Looking at the backend code I see there could be leading/trailing whitespace too, so perhaps we should use:

```js
const baseTitle = useContext(ConfigContext).baseTitle.trim();
```